website/archiver/utils.py

```diff
@@ -205,9 +206,14 @@ def get_file_map(node, file_map):
 
 def find_registration_file(value, node):
     from osf.models import AbstractNode as Node
-
     orig_sha256 = value['sha256']
-    orig_name = value['selectedFileName']
+    orig_name = sanitize.unescape_entities(
+        value['selectedFileName'],
+        {
```

Comment:
A small suggestion: for all uses of the function here and in tests, please include the keyword argument where appropriate. (eg `safe={}`)

Being explicit with kwargs always helps avoid future bugs (like if you added or changed params later), but this particular kwarg has a deeper meaning, and it's especially worth being explicit about the intended behavior in this case.